src/hub.js

```diff
@@ -1511,6 +1507,13 @@ document.addEventListener("DOMContentLoaded", async () => {
 
       await presenceSync.promise;
 
+      try {
+        // If scriptsPromise is null (i.e. we rejoined) then this is a no-op
+        await scriptsPromise;
```

Comment:
scriptsPromise will never be null

Reply:
`await scriptsPromise;` is fine, without the try/catch

Reply:
Oh, while scriptsPromise won't be null, we still want to try/catch if the importing fails. Just change your comment here :)